app/main.py

```diff
@@ -2,0 +1,10 @@
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
+    [elf.play_elf_song() for elf in elves]
+
+
+def feast_of_the_dwarves(dwarfs: list) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarfs]
```

Comment:
Using a list comprehension solely for its side effects is not recommended. Consider using a simple for loop instead: `for dwarf in dwarfs: dwarf.eat_favourite_dish()`.